lib/rototiller/task/params/option.rb

```diff
@@ -40,6 +40,11 @@ def add_argument(*args, &block)
       def to_str
         [@name.to_s, @arguments.to_s].compact.join(' ')
       end
+
+      def stop
+        return true if @arguments.stop?
+        return true unless @name
+      end
```

Comment:
ah.  so in commands it's really the params that stop.  because if they have a default and there is no env_var it continues.  got it.